src/DefaultTestReporter.js

```diff
@@ -132,13 +132,24 @@ function (config, aggregatedResults) {
     colors.GREEN + colors.BOLD
   );
 
-  var pluralTestSuites =
-      aggregatedResults.numTotalTestSuites === 1 ?
-      'test suite' : 'test suites';
+  if (aggregatedResults.numFailedToRunTestSuites) {
```

Comment:
@cpojer: I changed it to `numRuntimeErrorTestSuites`.

Do you already have some ideas about refactoring it?